addons/l10n_in/__manifest__.py

```diff
@@ -21,6 +21,7 @@
     'depends': [
         'account_tax_python',
         'base_vat',
+        'account_audit_trail',
```

Comment:
We need this module in 17.0, let's not change anything in `l10n_in` for now.
We will deal with the duplicate module later.

Basically, we should only add code in the PR, not even move files.

Reply:
@william-andre it's needed in any other country?
if not then it's okay to target the master.
or any specific goal to target 17.0?

Reply:
Yes, it is needed by end of year for Sweden.

Reply:
okay, perfect.
@kasp-odoo just add a new module not remove anything.